<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Зятьков ИДБ-21-10</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Препроцессор</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>691</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Стержни</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>192</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Длина(L)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Площадь(A)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Модуль 
упругости (E)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Допускаемое 
напряжение (σ)</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>101</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Опоры</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Заделка слева</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Заделка справа</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Заделка с обоих сторон</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>461</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Нагрузки на узлы</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>331</width>
         <height>181</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Узел</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Нагрузка (F)</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>111</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>pushButton_6</zorder>
       <zorder>pushButton_5</zorder>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>240</y>
        <width>461</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Нагрузки на стержни</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>331</width>
         <height>181</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Стержень</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Нагрузка (qx)</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>111</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Процессор</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>941</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Постпроцессор</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Таблицы</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>441</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Продольные силы Nx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>Перемещения Ux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>Нормальные напряжения σx</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>451</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Графики</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>431</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Продольные силы Nx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Перемещения Ux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Нормальные напряжения σx</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
